modules/utility/init.zsh

```diff
@@ -175,9 +175,13 @@ elif (( $+commands[wget] )); then
   alias get='wget --continue --progress=bar --timestamping'
 fi
 
-# Resource Usage
-if (( $+commands[pydf] )); then
-  alias df=pydf
+if zstyle -s ':prezto:module:utility' pydf 'yes'; then
```

Comment:
I think the `if zstyle -t ':prezto:module:utility:df' color` is the same as the original error, it doesn't make explicit what the option is doing. pydf is a partial reimplementation of df in python that has colors rather than a wrapper that just colors the output.